src/hmda-help/institution/index.js

```diff
@@ -65,6 +68,26 @@ class Institution extends Component {
     if (state && state.wasAddition) {
       this.setState({ wasAddition: state.wasAddition })
     }
+
+    // Feature: Direct linking
+    // Update URL with /update/institution/:id/:year
+    if (this.props.match.params.year && this.props.match.params.id && state) {
+      let splitURL = this.props.history.location.pathname.split('/')
+      // Contains lei that needs to be updated
+      splitURL[3] = state.institution.lei.toUpperCase()
+      this.props.history.push({
+        pathname: splitURL.join('/'),
+        state: { institution: state.institution }
+      })
+      // Contains year that needs to be updated
+      splitURL[4] = state.institution.activityYear.toString()
```

Comment:
Should this be above Line 78? `splitURL` is not used after this assignment is made.

Reply:
You are correct, must of accidentally moved that code below. Has been fixed!